crates/turborepo-lib/src/task_hash.rs

```diff
@@ -457,12 +457,16 @@ impl TaskHashTracker {
         state.package_task_outputs.insert(task_id, outputs);
     }
 
-    pub fn cache_status(&self, task_id: &TaskId) -> Option<CacheResponse> {
+    pub fn cache_status(&self, task_id: &TaskId) -> Option<CacheResult<CacheHitMetadata>> {
         let state = self.state.lock().expect("hash tracker mutex poisoned");
-        state.package_task_cache.get(task_id).copied()
+        state.package_task_cache.get(task_id).cloned()
```

Comment:
If we move this back to just `CacheHitMetadata` we can move it back to `copied()`